app/controllers/api/base_controller.rb

```diff
@@ -8,31 +8,45 @@ class BaseController < ApplicationController
 
     before_action :authenticate
     before_action :set_context_source
+    before_action :track_api_key_usage
     include Trackable
 
     rescue_from ActionController::ParameterMissing, with: :bad_request_error
 
     private
 
-    def authenticate
-      auth_header = request.headers['Authorization']
+    attr_reader :current_api_key, :current_organization
 
-      return unauthorized_error unless auth_header
+    def authenticate
+      return unauthorized_error unless auth_token
 
-      api_key = auth_header.split(' ').second
+      @current_api_key = ApiKey.active.find_by(value: auth_token)
```

Comment:
I think you'll tackle it in a future PR, but `.active` imho should be a default scope

Reply:
Exactly.